.github/workflows/pr-checks-for-qa-approvals.yml

```diff
@@ -14,6 +14,7 @@ on:
       - 'VAMobile/package.json'
       - 'VAMobile/.detoxrc.json'
       - '!VAMobile/src/*.test.tsx'
+      - '!.github/**'
```

Comment:
Is this change meant to be in this PR? If so, in the future please mention it in the description or leave a comment for the reviewer about it since it seems to be unrelated to the ticket.

Reply:
Let me create PR for it.